docs/src/main/asciidoc/spring-cloud-sleuth.adoc

```diff
@@ -12,299 +12,41 @@ include::intro.adoc[]
 
 include::features.adoc[]
 
-=== Introduction to Brave
+== Introduction to Brave
 
-IMPORTANT: Starting with version `2.0.0`, Spring Cloud Sleuth uses
-https://github.com/openzipkin/brave[Brave] as the tracing library.
-For your convenience, we embed part of the Brave's docs here.
+Brave is a distributed tracing instrumentation library. Brave typically
+intercepts production requests to gather timing data, correlate and propagate
+trace contexts. While typically trace data is sent to Zipkin server,
+third-party plugins are available to send to alternate services such as Amazon
+X-Ray.
 
-IMPORTANT: In the vast majority of cases you need to just use the `Tracer`
-or `SpanCustomizer` beans from Brave that Sleuth provides. The documentation below contains
-a high overview of what Brave is and how it works.
+Spring Cloud Sleuth is a layer over https://github.com/openzipkin/brave[Brave].
```

Comment:
Nit: might be better to make the first mention of Brave (in the previous paragraph) the link.